### PR TITLE
Add descriptions to all jinja page s

### LINKED ISSUE #3061: Add contents in dbt Jinja functions page
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Discussion with @b-per to improve this page and provide more contextual information for each jinja function
Screenshot 2023-03-23 at 13 51 31

Per discussion with @runleonarun , this page is [autogenerated](https://docusaurus.io/docs/sidebar/autogenerated)

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/reference/dbt-jinja-functions

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
Discussed with @b-per , adding clear jinja function descriptions to improve user reading experience. There is limitation with what to do with autogenerated texts per [slack thread](https://dbt-labs.slack.com/archives/C01C2H652BC/p1679671378950009).

Closes #3061

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @b-per, i've tried adding clearer descriptions so users can understand the context of the page when they land on the jinja functions page. do you mind taking a look to see if my summary is accurate?

File: website/docs/reference/dbt-jinja-functions/as_native.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "as_native"
 id: "as_native"
+description: "Use this filter to coerce jinja-compiled output into its native python."
```
Comment (matthewshaver) [inline website/docs/reference/dbt-jinja-functions/as_native.md]:
```suggestion
description: "Use this filter to coerce Jinja-compiled output into its native python."
```

**Comment by mirnawong1** [comment]
thanks so much @matthewshaver - committed your suggestion and let me know if it's good to go 😄

**Review by runleonarun — APPROVED** [review]
Looks good to me!

**Pull request merged**